indico/types/submission.py

```diff
@@ -0,0 +1,24 @@
+from indico.types import BaseType
+
+
+class Submission(BaseType):
+    """
+    A Submission in the Indico Platform
+
+    Attributes:
+        id (int): the Submission id
+        dataset_id (int): the Dataset id
+        workflow_id (int): the Workflow id
+        status (str): status of the submission
+        input_file (str): local url to stored input
+        input_filename (str): name of the original file
+        result_file (str): local url to the stored output
```

Comment:
Using previous comments as a guide, I think Indico URLs should be described as
> URL of the datafile within the Indico Platform

https://github.com/IndicoDataSolutions/indico-client-python/blob/master/indico/types/datafile.py#L14

Reply:
good comment, i had pulled these from Fog, but there's probably a more informative description to surface for SDK users.